### PR TITLE
add Elrond support

### PR DESCRIPTION
**Pull request by aranhaagency**
done https://github.com/ensdomains/address-encoder/issues/53
ref: https://github.com/trustwallet/wallet-core/blob/8d3100f61e36d1e928ed1dea60ff7554bba0db16/tests/Elrond/AddressTests.cpp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ This library currently supports the following cryptocurrencies and address forma
  - XRP (base58check-ripple)
  - BCH (base58check and cashAddr; decodes to cashAddr)
  - BNB (bech32)
+ - EGLD (bech32)
```
Comment (makoto) [inline README.md:43]: Can you provide reference link so that we can verify?

Comment (aranhaagency) [reply README.md:43]: @makoto https://github.com/trustwallet/wallet-core/blob/8d3100f61e36d1e928ed1dea60ff7554bba0db16/tests/Elrond/AddressTests.cpp

Comment (makoto) [reply README.md:43]: ah sorry, I missed out your initial comment

**Review by Arachnid — APPROVED** [review]

**Comment by Arachnid** [comment]
Can you please fix the merge conflicts?

**Pull request merged**